app/javascript/gobierto_investments/webapp/components/Map.vue

```diff
@@ -73,7 +80,8 @@ export default {
       geojsons: [],
       geojsonOptions: {},
       item: null,
-      center: [40.199867, -4.0654947] // Spain center
+      center: [40.199867, -4.0654947], // Spain center
+      titleButton: 'Tour Virtual'
```

Comment:
Could you translate this string too?

Reply:
We need an array with coordinates[lat/long] @Crashillo